common/models/service-instance.json

```diff
@@ -24,7 +24,12 @@
     },
     "startTime": {
       "type": "date",
-      "description": "The ID of the service group this executor belongs to"
+      "description": "Timestamp when instance was started"
+    },
+    "stopTime": {
+      "type": "date",
+      "description": "Timestamp when instance was stopped (cloud usecases)",
+      "default": "1/1/1970 00:00:00 GMT+0"
```

Comment:
That's unfortunate.. can we compromise on at least using a real [date format](https://xkcd.com/1179/)?

Should be `1970-01-01T00:00:00.000Z`, and I propose that any datastore that chokes on that doesn't meet our minimum requirements.